v3/util/time.go

```diff
@@ -37,6 +37,7 @@ var (
 	RFC4630Date                = time.Date(2006, time.August, 1, 0, 0, 0, 0, time.UTC)
 	RFC5280Date                = time.Date(2008, time.May, 1, 0, 0, 0, 0, time.UTC)
 	RFC6818Date                = time.Date(2013, time.January, 1, 0, 0, 0, 0, time.UTC)
+	RFC6962Date                = time.Date(2013, time.June, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
Comment on this already made in #841. So long as both go in, then it doesn't really matter which has the infra changes.